File: program/convert-minutes-to-weeks/ConvertMinutesToWeeks.ts
function ConvertMinutesToWeeks(minutes: number) {
    // 1 day = 1440 minutes
    // 1 week = 10080 minutes

    const days = minutes / 1440;
    const week = days / 7;
    return week;

}

console.log(ConvertMinutesToWeeks(50400) + " weeks");

File: program/program/convert-minutes-to-weeks/ConvertMinutesToWeeks.kt
fun main() {
    print("Enter minutes : ")
    println()
    var input = readLine()

    try {
        println("$input minutes = " + ConvertMinutesToWeeks(input!!.toDouble()) + " weeks")
    } catch (exception: Exception) {
        println("Input string was not in a correct format")
    }

}

private fun ConvertMinutesToWeeks(minutes: Double): Double {
    val days = minutes / 1440;
    val week = days / 7;

    return  week
}